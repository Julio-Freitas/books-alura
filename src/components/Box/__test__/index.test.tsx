import { describe, expect, it } from "vitest";
import { render, screen } from "@testing-library/react";
import { Box } from "..";

describe("Box", () => {
  it("Should be show title", () => {
    const {container}=render(<Box title="Title" />);
    expect(screen.getByRole('heading', {name:/title/i})).toBeInTheDocument();
    expect(container.firstChild).toMatchSnapshot()
  });
});
