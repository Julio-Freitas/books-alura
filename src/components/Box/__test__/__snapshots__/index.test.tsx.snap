// Vitest Snapshot v1

exports[`Box > Should be show title 1`] = `
<section
  class="sc-fnGiBr hSOubj"
>
  <h2>
    Title
  </h2>
  <section
    class="sc-gswNZR PXctC"
  />
</section>
`;
